test/bats/helpers/lxly-bridge.bash

```diff
@@ -89,6 +89,55 @@ function bridge_asset() {
     fi
 }
 
+function get_bridges() {
+    local aggkit_node_url=$1
+    local l1_rpc_network_id=$2
+
+    bridges_result=$(cast rpc --rpc-url "$aggkit_node_url" "bridge_getBridges" "$l1_rpc_network_id")
+
+    echo "------ bridges_result ------"
+    echo "$bridges_result"
+    echo "------ bridges_result ------"
+
+    count=$(jq -r '.count' <<< "$bridges_result")
+    if [ "$count" -ne 1 ]; then
+        echo "Assertion failed. 'count' should be 1. Actual: $count"
+        exit 1
+    fi
```

Comment:
I'd say, let's search for a specific `bridge_tx_hash` and return it instead. If it is not found, then return an error. 

The reason I'm saying this is that then we could reuse this function in other tests as well (imagine running multiple tests against the same cluster, which is the case on the CI, and therefore there could be more than one bridges). WDYT?